pkgs/applications/networking/browsers/chromium/source/default.nix

```diff
@@ -16,6 +17,11 @@ let
     "s,$main/third_party(/.*)?$,$bundled\\1,"
     "s,^/,,"
   ]);
+  wideVinePatch = fetchurl {
+    url = "https://projects.archlinux.org/svntogit/packages.git/plain/trunk/chromium-widevine.patch?h=packages/chromium";
```

Comment:
I would strongly advise against linking to the trunk of a remote svntogit web UI and expecting that it won't change.  This is such a tiny patch, too - why not just save it in the nixpkgs tree along with the other patches referenced in this file?